coding/python_style_guide.rst

```diff
@@ -582,15 +582,13 @@ Within a module, follow the order:
 1. Shebang line, ``#! /usr/bin/env python`` (only for executable scripts)
 2. Module-level comments (such as the `license statement <https://github.com/lsst/templates/blob/master/CopyrightHeader.py>`__)
 3. Module-level docstring
-4. Imports
-5. ``__all__`` statement, if any
-6. Private module variables (names start with underscore)
-7. Private module functions and classes (names start with underscore)
-8. Public module variables
-9. Public functions and classes
-10. Optional test suites
-
-.. FIXME JSick: Delete mention of test suites?
+4. ``from __future__ import absolute_import, division, print_function``
+5. ``__all__ = [...]`` statement, if present
+6. Imports
```

Comment:
Do you think this is clear enough, or would it be better to say "Other imports" or something like that?